client/my-sites/subscribers/components/subscribers-header/subscribers-header.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button, Gridicon } from '@automattic/components';
+import { HelpCenter } from '@automattic/data-stores';
+import { useLocalizeUrl } from '@automattic/i18n-utils';
+import { useDispatch as useDataStoreDispatch } from '@wordpress/data';
+import { translate } from 'i18n-calypso';
+import { ReactElement } from 'react';
+import { navItems } from 'calypso/blocks/stats-navigation/constants';
+import NavigationHeader from 'calypso/components/navigation-header';
+import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
+import { useSubscribersPage } from 'calypso/my-sites/subscribers/components/subscribers-page/subscribers-page-context';
+import { useSelector } from 'calypso/state';
+import getIsSiteWPCOM from 'calypso/state/selectors/is-site-wpcom';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import { SubscribersHeaderPopover } from '../subscribers-header-popover';
+
+type SubscribersHeaderProps = {
+	selectedSiteId: number | undefined;
+	disableCta: boolean;
+	hideSubtitle?: boolean;
+};
+
+export const SubscribersHeader = ( {
+	selectedSiteId,
+	disableCta,
+	hideSubtitle,
+}: SubscribersHeaderProps ): ReactElement => {
+	const HELP_CENTER_STORE = HelpCenter.register();
+	const { setShowAddSubscribersModal } = useSubscribersPage();
+	const localizeUrl = useLocalizeUrl();
+	const { setShowHelpCenter, setShowSupportDoc } = useDataStoreDispatch( HELP_CENTER_STORE );
+	const selectedSite = useSelector( getSelectedSite );
+	const siteId = selectedSite?.ID || null;
+	const isWPCOMSite = useSelector( ( state ) => getIsSiteWPCOM( state, siteId ) );
+
+	const openHelpCenter = () => {
+		setShowHelpCenter( true );
+		setShowSupportDoc( localizeUrl( 'https://wordpress.com/support/paid-newsletters/' ), 168381 );
```

Comment:
This localize_url is redundant. We already localize every URL. You can also ditch the post ID, it is not used anymore. We also don't need to open the HelpCenter, that is covered when calling show Support Doc. Help Center is my last product haha.

I know this isn't yours per-se since you copied it. Just figured we could clean it up a little.